api/flickr.py

```diff
@@ -1,11 +1,60 @@
-# get images from flickr
-from PIL import Image
+import requests
+import os
 
-def get_image(search_term):
 
-    # talk to flickr
-    # download image 
-    # save, create Image to disk
-    return Image.open('api/broccoli.png')
+def getImage():
+    item = input("Enter the food item you would like an image for: ")
```

Comment:
@abdala2020 @7ibakana  I think all the user prompt for user input should be from ui.py, what do you think? This flickr.py should only be making the call to the api.